bin/bank/pycbc_brute_bank

```diff
@@ -89,6 +89,17 @@ if args.input_config is not None:
 
     dists_joint = prior_from_config(cp=config_parser)
 
+if not args.tau0_end:
+    """
+    If tau0_end is not provided, it is derived and rounded up to the
+    upper tens.
+    This is necessary to cover up all the phase space.  
+    """
+    import math
+    from pycbc.conversions import tau0_from_mass1_mass2
+    args.tau0_end = math.ceil(tau0_from_mass1_mass2(args.min[0], args.min[1],
```

Comment:
The problem here is that this assumes that m1/m2 are actually in these arguments, but that is not necessarily the case. In fact, it may no be provided on the command line at all.

Reply:
Ok, I removed this part of the code. However, does it make sense to keep a similar function that could read all the possible options? This way, we wouldn't have to manually derive the tau0-end (which I believe is mandatory in the code, am I correct?), and it would cover the entire phase space.

Reply:
@MarcoCusinato This would be a nice feature, but I think it is somewhat more complicated that you might imagine, given that one can use somewhat arbitrary coordinates. Coordinates have to be convertible to m1/m2 at a certain point, but there doesn't have to be a simple boundary. The best I can think to suggest if you want it to be generic is to find the min/ max by sampling from the distribution. 

If you think this is reasonable, maybe you could add this in a separate PR after this is merged?

Reply:
You can pass a prior config file (the same format we use for parameter estimation). That also means you can use things like arbitrary coordinates (as long as you provide the transform back to the ones used for waveform generation) in addition to constraints, etc. It also allows you some control over how proposal points are selected as it will periodically draw from the provided prior in addition to using a kde of previously accepted values. (if you've seen the output, this is the difference between a 'round' marked with 'U' (prior) and 'K' (kde).

Reply:
A trivial example would be 
```
[variable_params]
mass1 =
mass2 =

[static_params]

[prior-mass1]
name = uniform
min-mass1 = 10.
max-mass1 = 80.

[prior-mass2]
name = uniform
min-mass2 = 10.
max-mass2 = 80.
```

But there are of course many other priors available, and you can provide a [waveform_transform-*] section and constraint sections. See some PE examples for uses of this, to do coordinate transforms from whatever coordinates you want. 

https://pycbc.org/pycbc/latest/html/inference.html#waveform-transforms

https://pycbc.org/pycbc/latest/html/inference.html#constraints

Reply:
Thank you very much!